src/tdlib/chat.rs

```diff
@@ -231,6 +243,38 @@ mod imp {
             PROPERTIES.as_ref()
         }
 
+        fn set_property(
+            &self,
+            obj: &Self::Type,
+            _id: usize,
+            value: &glib::Value,
+            pspec: &glib::ParamSpec,
+        ) {
+            match pspec.name() {
+                "id" => self.id.set(value.get().unwrap()),
+                "type" => self.type_.set(value.get().unwrap()).unwrap(),
+                "is-blocked" => obj.set_is_blocked(value.get().unwrap()),
+                "title" => {
+                    obj.set_title(value.get::<Option<String>>().unwrap().unwrap_or_default())
+                }
+                "avatar" => obj.set_avatar(value.get().unwrap()),
+                "last-read-outbox-message-id" => {
+                    obj.set_last_read_outbox_message_id(value.get().unwrap());
+                }
+                "last-message" => obj.set_last_message(value.get().unwrap()),
+                "order" => obj.set_order(value.get().unwrap()),
+                "is-pinned" => obj.set_is_pinned(value.get().unwrap()),
+                "unread-mention-count" => obj.set_unread_mention_count(value.get().unwrap()),
+                "unread-count" => obj.set_unread_count(value.get().unwrap()),
+                "online-member-count" => obj.set_online_member_count(value.get().unwrap()),
+                "draft-message" => obj.set_draft_message(value.get().unwrap()),
+                "notification-settings" => obj.set_notification_settings(value.get().unwrap()),
+                "permissions" => obj.set_permissions(value.get().unwrap()),
+                "session" => self.session.set(Some(&value.get().unwrap())),
+                _ => unimplemented!(),
+            }
+        }
+
```

Comment:
This should not be here, because we made all properties read-only some time ago. I think you committed it by accident.